ludwig/schema/optimizers.py

```diff
@@ -354,7 +355,12 @@ class GradientClippingConfig(BaseMarshmallowConfig):
     clipvalue: Optional[float] = FloatRange(default=None, allow_none=True, description="")
 
 
-def GradientClippingDataclassField(default={}, allow_none=True, description="TODO"):
+def GradientClippingDataclassField(
+    default={},
+    allow_none=True,
+    description="Parameters for gradient clipping.",
+    parameter_metadata: ParameterMetadata = None,
```

Comment:
I would not let `parameter_metadata` be set here, and just have it set to `TRAINER_METADATA['gradient_clipping']`. Actually I'd remove all the parameters here, being only used in the trainer it's something that should not have open arguments.

Reply:
Done, though I left `description` and `default` since I think it's still visually useful for those fields to appear in the Trainer schema.